packages/sandstone/template/package.json

```diff
@@ -37,7 +37,7 @@
     "@enact/ui": "^4.0.0",
     "ilib": "^14.6.0",
     "prop-types": "^15.6.2",
```

Comment:
Would you please update other dependencies to up to date as well?

Reply:
including enact packages

Reply:
OK